build/quicklink.php

```diff
@@ -31,13 +31,13 @@ function quicklink_enqueue_scripts() {
 		return;
 	}
 
-	wp_enqueue_script( 'quicklink', QUICKLINK_URL . 'quicklink.min.js', array(), '<##= pkg.version ##>', true );
+	wp_enqueue_script( 'quicklink' );
```

Comment:
Apparently this is needed to make it actually get added to the footer:

```suggestion
	wp_enqueue_script( 'quicklink', '', array(), false, true );
```

It seems passing `true` to `WP_Scripts::add` isn't working.